src/widgets/JSONForm/JSONForm.js

```diff
@@ -207,6 +207,16 @@ export const JSONFormComponent = React.forwardRef(
       []
     );
 
+    const mounted = useIsMounted();
+    const [noValidate, setNoValidate] = React.useState(true);
+    const touchedFields = {};
+
+    React.useEffect(() => {
```

Comment:
not strictly necessary now that touched status is being used - I thought good to reduce the number of times that validation is called so left it in